ci-operator/step-registry/observers/ocp-baremetal-qe/observers-ocp-baremetal-qe-commands.sh

```diff
@@ -0,0 +1,574 @@
+#!/bin/bash
+
+# shellcheck disable=SC2154
```

Comment:
Can we avoid suppressing them globally and even better supporting them by fixing the code? When they are really exception, we should comment and say why we are not respecting those rules

Reply:
sure, I was testing global suppression for the exceptions like`. <(echo "$bmhost")` , I will fix the code